src/1_create_canonical.jl

```diff
@@ -131,8 +200,46 @@ output_features.ReefMod_area_m2 .= rme_ids[:, :area_km2] .* 1e6
 # Calculate `k` area (1.0 - "ungrazable" area)
 output_features.ReefMod_habitable_proportion .= 1.0 .- rme_ids[:, :sand_proportion]
 
+
+
 # Reproject output_features from GDA94 EPSG4283 to GDA2020 EPSG7844 to match GBRMPA geohub data
-output_features.geometry = AG.reproject(output_features.geometry, GI.crs(output_features[1,:geometry]), EPSG(7844); order=:trad)
+# THIS WAS CAUSING AN ISSUE DO TO THE MIX OF POLYGON AND MULTIPOLYGON GEOMETRIES NEW SECTION BELOW
+#output_features.geometry = AG.reproject(output_features.geometry, GI.crs(output_features[1,:geometry]), EPSG(7844); order=:trad)
```

Comment:
Code is version controlled.

It's fine to delete the line and explain in the commit message why things are removed/changed.

Otherwise the entire file will be filled with commented lines of code.